### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Technologx**
Added the community and my wallpaper app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,10 +74,11 @@ Want your app listed here? Mention me at [Google Plus](https://plus.google.com/u
 * [myWalls](https://play.google.com/store/apps/details?id=com.lc.mywalls)
 * [NEVER SETTLE Wallpapers](https://play.google.com/store/apps/details?id=com.wilfredparmar.never.settle)
 * [Wallerta](https://play.google.com/store/apps/details?id=com.xorrisont.wallerta)
+* [Sulox](https://goo.gl/EcrGN9)
 
 # License
 ```
-Copyright (c) 2017 Dani Mahardhika
+Copyright © Dani Mahardhika
```
Comment (danimahardhika) [inline README.md]: Do not change the license part

File: README.md [hunk 2]
```diff
@@ -62,6 +62,9 @@ Take a look inside <a href="https://github.com/danimahardhika/wallpaperboard/wik
 # Support Development
 Support development by making donation through demo app at Google Play.
 
+# Support for the Dashboard or Suggestions
+[WallpaperBoard Community](https://goo.gl/R2RjwM)
```
Comment (danimahardhika) [inline README.md]: Title and description should match with the one shown on CandyBar readme

Comment (Technologx) [inline README.md]: Feel free to change it to what you need to change it to.

File: README.md (hunk 1 above)
Comment (Technologx) [inline README.md]: Alright I was just trying to help with the symbol

Comment (danimahardhika) [reply README.md]: That was a part of the license, do not change it

Comment (Technologx) [inline README.md]: I won't I'm sorry

**Pull request closed without merging**